lang/de_DE.po

```diff
@@ -284,10 +284,8 @@ msgstr ""
 msgid "%s made unknown change"
 msgstr "%s machte unbekannte Änderungen"
 
-#
-# File: src/etc/inc/crypt.inc, line: 47
```

Comment:
I don't like this stripping. I've merged it in, but the German file lost a lot of references to lines in the first commit already. Is this poedit doing something nasty?